nautobot/core/ui/object_detail.py

```diff
@@ -395,14 +396,20 @@ class ObjectsTablePanel(Panel):
     def __init__(
         self,
         *,
-        table_key,
+        table_class,
+        table_filter=None,
+        table_attribute=None,
+        select_related_fields=None,
+        prefetch_related_fields=None,
+        order_by_fields=None,
         table_title=None,
         max_display_count=None,
         include_fields=None,
         exclude_fields=None,
```

Comment:
Not critical for this PR but we should consider renaming these to `include_columns` and `exclude_columns` instead?